### DISCUSSION TITLE
Dependency updates

### DISCUSSION THREAD

**Discussion by drewhamilton** [Backbase/DeferredResources#85] [General] [answered]
Deferred Resources last shipped over a year ago and many of its dependencies are out of date. I'd like to update them, but first I'd like to understand the potential impact on consumers (including Backbase).

Updates I'd like to make include:
* **Kotlin:** 1.5.21 → 1.7.20
* **Compose compiler:** 1.0.1 → 1.3.2
* **Compose runtime:** 1.0.1 → 1.3.0
* **Lottie:** 4.1.0 → 5.2.0 (I believe this has breaking changes, so may require a new module)

Are any consumers not already using these updated versions in their consuming apps/libraries? Do you expect any problems with these upgrades?

There are also tooling updates, like AGP and Poko, but these wouldn't affect consumers.

**Comment by donaldchickeme** [comment, marked as answer]
There is no real impact on Backbase as far as I can tell. Releasing new versions of the main lib and the adapters with these upgrades should be just fine.
